extensions/notion/src/utils/notion/page.ts

```diff
@@ -69,6 +112,7 @@ export async function search(query?: string, nextCursor?: string) {
       direction: "descending",
       timestamp: "last_edited_time",
     },
+    filter: { property: "object", value: "page" },
```

Comment:
add page filter, @bkeys818 @thomaslombart  please  help review is this logic make sense.
In the "append text to page" scene, according to our ui form field, I think we are not will to add text to dataPage table?